src/System.Private.ServiceModel/src/System/IdentityModel/SecurityUtils.cs

```diff
@@ -209,6 +211,63 @@ internal static IIdentity CloneIdentityIfNecessary(IIdentity identity)
             return identity;
         }
 
+#if !FEATURE_NETNATIVE // NegotiateStream
+        /// <SecurityNote>
+        /// Critical - calls two critical methods: UnsafeGetWindowsIdentityToken and UnsafeCreateWindowsIdentityFromToken
+        /// Safe - "clone" operation is considered safe despite using WindowsIdentity IntPtr token
+        ///        must not let IntPtr token leak in or out
+        /// </SecurityNote>
+        [SecuritySafeCritical]
+        internal static WindowsIdentity CloneWindowsIdentityIfNecessary(WindowsIdentity wid)
```

Comment:
Is [SecuritySafeCritical] even needed now?  I think we've been removing them over time.